src/core_modules/capture-core/components/D2Form/D2Form.component.js

```diff
@@ -110,9 +110,11 @@ class D2Form extends React.PureComponent<PropsForPureComponent> {
             key={section.id}
         >
             <D2SectionContainer
-                innerRef={(sectionInstance) => { this.setSectionInstance(sectionInstance, section.id); }}
+                innerRef={(sectionInstance) => {
+                    this.setSectionInstance(sectionInstance, section.id);
+                }}
                 sectionMetaData={section}
-                customForm={this.props.formFoundation.customForm}
+                customForm={section.id === '#MAIN#' ? this.props.formFoundation.customForm : null}
```

Comment:
Do you think that skipping the ternary operator here will be fine? 
```suggestion
                customForm={this.props.formFoundation.customForm}
```

Reply:
The ternary operator is there in case the enrollment form has a `Tracked Entity Feature Type`. If there is a Feature type, the form will render a separate section. If we don't use the ternary operator, the form will change the html of all available sections in the formFoundation, including the section for Feature Type. (And any other sections if we ever have to implement it in the future). 
This way, it will only replace the HTML in the `#MAIN#`-section.
Let me know if this is unclear or if I should still make any changes.

Reply:
Ah, I see. Follow-up suggestion: Can customForm be a property of the Section object instead of the RenderFoundation object?

Reply:
Good idea 👍 Implemented!